presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -163,7 +164,7 @@ public class HiveClientConfig
     private boolean parquetPushdownFilterEnabled;
     private boolean adaptiveFilterReorderingEnabled = true;
     private Duration fileStatusCacheExpireAfterWrite = new Duration(0, TimeUnit.SECONDS);
-    private long fileStatusCacheMaxSize;
+    private DataSize fileStatusCacheMaxSize = new DataSize(1, GIGABYTE);
```

Comment:
Why are we keeping the default value as 1 GB?

Reply:
changed it to 0.